src/validators/isGeometryBox.js

```diff
@@ -0,0 +1,36 @@
+import {walk} from 'postcss-value-parser';
+import globals from '../util/globals';
```

Comment:
Globals don't need to be handled in any sub-validator (as they are global keywords).